src/realm/exec/realm_trawler.cpp

```diff
@@ -246,9 +247,15 @@ class Table : public Array {
             m_column_types.init(alloc, spec.get_ref(0));
             m_column_names.init(alloc, spec.get_ref(1));
             m_column_attributes.init(alloc, spec.get_ref(2));
-            if (spec.size() > 3) {
+            if (spec.size() > 5) {
```

Comment:
Please add either indicative names of these constants or comments about the intent of these comparisons against sizes of the spec.

Reply:
I think it is obvious that the spec array has to have a certain size if the initializations can be made. The names of the variables should indicate the use. I have added a few comments, though.